pkg/tsdb/elasticsearch/time_series_query.go

```diff
@@ -508,3 +509,15 @@ func (p *timeSeriesQueryParser) parseMetrics(model *simplejson.Json) ([]*MetricA
 	}
 	return result, nil
 }
+
+func getPipelineAggField(m *MetricAgg) string {
+	// In frontend we are using "Field" as pipelineAggField
+	// In backend were originally using "PipelineAggregate" as pipelineAggField
+	// Therefore we are going to check Field first and then PipelineAggregate to ensure that we are not breaking anything
```

Comment:
There might be historical reason why we do this. 
```suggestion
	// In frontend we are using "Field" as pipelineAggField. There might be historical reason why in 
	// backend we were using  "PipelineAggregate" as pipelineAggField, so for now let's 
	// check Field first and then PipelineAggregate to ensure that we are not breaking anything
	// TODO: Investigate, if we can remove check for "PipelineAggregate"
```